index.html

```diff
@@ -0,0 +1,327 @@
+          >
+            <img src="img/icons/conversation.png" alt="Believers_meeting" />
+            <h3 class="text-center sp-t">Believers Meeting</h3>
+            <p>Each one has a tongue, interpreatations.</p>
+          </div>
+          <div
+            class="
+              d-flex
+              my-1
+              event-bg
+              justify-content-around justify-content-md-center
+              align-items-center
+              flex-md-column
+              col-12
+              text-center
+            "
+          >
+            <img src="img/icons/pray.png" alt="pray" />
+            <h3>Prayer</h3>
+            <p>Heartfelt prayer.</p>
+          </div>
+          <div
+            class="
+              d-flex
+              my-1
+              event-bg
+              justify-content-around justify-content-md-center
+              align-items-center
+              flex-md-column
+              col-12
+              text-center
+            "
+          >
+            <img src="img/icons/worship-icon.png" alt="worship" />
+            <h3>Worship</h3>
+            <p>Worship in truth and spirit.</p>
+          </div>
+        </div>
+        <div
+          class="see-height d-flex justify-content-center align-items-center"
+        >
+          <button class="d-lg-none bg-join">
+            Join L2 Ministries Conference 2021
+          </button>
+          <p class="under-p d-none d-lg-block">SEE THE WHOLE PROGRAM</p>
+        </div>
+      </section>
+    </div>
+
+    <div class="container">
+      <section class="speakers">
+        <div class="my-5">
+
+       
+        <h3 class="text-center">Featured Speakers</h3>
+        <div class=" underline">
+          <hr >
+        </div>
+      </div>
+        <div class="row">
+          <section class="col-md-6 d-flex f-speaker mb-5">
+            <div class="img">
+
+            </div>
+            <div class="d-flex flex-column  ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+            
+          </section>
+          <section class="col-md-6 d-flex   f-speaker mb-5">
+             <div class="img">
+              
+            </div>
+            <div class="d-flex flex-column  ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+          </section>
+        </div>
+        <div class="row">
+          <section class="col-md-6 d-flex  f-speaker mb-5">
+            <div class="img">
+              
+            </div>
+            <div class="d-flex flex-column ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+          </section>
+          <section class="col-md-6 d-flex  f-speaker mb-5">
+            <div class="img">
+              
+            </div>
+            <div class="d-flex flex-column  ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+          </section>
```

Comment:
The articles section has 6 articles, the container for the speaker should be a `section` tag and each speaker item an `article` tag

- [ ] Kindly update this section with the required number of articles
- [ ] Kindly use semantic HTML tags

Reply:
Thanks but I believe we were to make this implementation as personal as possible, not just a dummy copy of template